1-org/envs/shared/iam.tf

```diff
@@ -162,21 +162,21 @@ resource "google_project_iam_member" "audit_bq_data_viewer" {
 }
 
 resource "google_organization_iam_member" "org_scc_admin" {
-  count  = var.gcp_groups.scc_admin != null && local.parent_folder == "" && var.enable_scc_notifications ? 1 : 0
+  count  = var.gcp_groups.scc_admin != null && local.parent_folder == "" ? 1 : 0
   org_id = local.org_id
   role   = "roles/securitycenter.adminEditor"
   member = "group:${var.gcp_groups.scc_admin}"
 }
 
 resource "google_project_iam_member" "project_scc_admin" {
-  count   = var.gcp_groups.scc_admin != null && var.enable_scc_notifications ? 1 : 0
+  count   = var.gcp_groups.scc_admin != null && var.enable_scc_resources_in_terraform ? 1 : 0
   project = module.scc_notifications[0].project_id
```

Comment:
There are still 3 references to `module.scc_notifications[0].project_id` that will throw an error now that we removed the count argument from the module `scc_notifications` in projects.tf. Revert these 3 references back to `module.scc_notifications.project_id`.

Somehow this passed Lint tests (can't explain why, though? This definitely throws an invalid index error when running Terraform plan in my local environment). I've kicked off the full CI build, takes some hours but I expect it will fail with the same invalid index error.